build/web/glaemscribe/js/charsets/eldamar.cst.js

```diff
@@ -0,0 +1 @@
  TW_74 ALDA \n\n\\char e024  TW_81 SILME \n\\** Unfortunately, monotengwar is missing silme nuquerna used for y in beleriand **\\\n\\char e025  TW_82 SILME_NUQUERNA SILME_NUQUERNA_ALT\n\\char e026  TW_83 ESSE\n\\char e027  TW_84 ESSE_NUQUERNA\n\n\\char e028  TW_91 HYARMEN \n\\char e029  TW_92 HWESTA_SINDARINWA \n\\char e02A  TW_93 YANTA\n\\char e02B  TW_94 URE\n\n\\char e02C  ARA\n\\char e02D  HALLA\n\\char e02E  TELCO\n\\char e02F  ?      \n\n\\char e030  REVERSED_OSSE\n\\char e031  BOMBADIL_W \n\\char e032  OSSE\n\\char e033  ?\n\n\\char e034  LIGATING_SHORT_CARRIER\n\\char e035  ?\n\\char e036  ANNA_OPEN\n\\char e037  CHRISTOPHER_QU\n\n\\char e038  ?\n\\char e039  BOMBADIL_HW\n\\char e03A  TW_MH MALTA_W_HOOK \n\\char e03B  TW_MH_BELERIANDIC VALA_W_HOOK \n\n\\char e03C  TW_HW_LOWDHAM HARP_SHAPED\n\\char e03D  VAIA WAIA VAIYA\n\\char e03E  ?\n\\char e03F  ?      \n\n\\char e040  A_TEHTA\n\\char e041  A_TEHTA_INF\n\\char e042  I_TEHTA_DOUBLE Y_TEHTA\n\\char e043  I_TEHTA_DOUBLE_INF PALATAL_SIGN Y_TEHTA_INF \n\n\\char e044  I_TEHTA\n\\char e045  I_TEHTA_INF NO_VOWEL_DOT UNUTIXE\n\\char e046  E_TEHTA\n\\char e047  E_TEHTA_INF\n\n\\char e048  E_TEHTA_DOUBLE\n\\char e049  E_TEHTA_DOUBLE_INF GEMINATE_DOUBLE\n\\char e04A  O_TEHTA\n\\char e04B  O_TEHTA_INF\n\n\\char e04C  U_TEHTA\n\\char e04D  U_TEHTA_INF\n\\char e04E  O_TEHTA_DOUBLE\n\\char e04F  U_TEHTA_DOUBLE\n\n\\char e050  NASALIZE_SIGN NASALIZE_SIGN_TILD      \n\\char e051  GEMINATE_SIGN GEMINATE_SIGN_TILD\n\\char e052  WA_TEHTA SEV_TEHTA\n\\char e053  TEHTA_BREVE A_TEHTA_CIRCUM_REVERSED\n\n\\char e054  E_TEHTA_GRAVE\n\\char e055  A_TEHTA_CIRCUM  \n\\char e056  A_TEHTA_REVERSED A_TEHTA_DOUBLE\n\\char e057  THINNAS  THINF_STROKE\n\n\\** THE TWO FOLLOWING ONES ARE HIGHLY DISCUSSABLE, MAYBE WE SHOULD ADD A VIRTUAL CHAR FOR THESE **\\\n\\char e058  SARINCE_ENDING_LONG SARINCE_FLOURISHED\n\\char e059  SARINCE COMBINING_SARINCE SHOOK_RIGHT_L SHOOK_LEFT_L \n\\char e05A  ?\n\\char e05B  ?\n\n\\char e065  PUNCT_EXCLAM\n\\char e066  PUNCT_INTERR\n\\char e067  ELVISH_PAREN PUNCT_PAREN_L PUNCT_PAREN_R PUNCT_PAREN_L_ALT PUNCT_PAREN_R_ALT BOOKMARK_SIGN\n\\char e068  PUNCT_TILD\n \n\\char e069  PUNCT_DTILD RING_MARK_L RING_MARK_R\n\n\\char e06A  DQUOT_OPEN\n\\char e06B  DQUOT_CLOSE\n\\char e06C  THORIN_EXCLAMATION\n\n\\char e070 NUM_0 \n\\char e071 NUM_1 \n\\char e072 NUM_2 \n\\char e073 NUM_3 \n\\char e074 NUM_4 \n\\char e075 NUM_5 \n\\char e076 NUM_6 \n\\char e077 NUM_7 \n\\char e078 NUM_8 \n\\char e079 NUM_9 \n\\char e07A NUM_10\n\\char e07B NUM_11\n\\char e07C NUM_12\n\n\\char e07D CIRC_TEHTA_INF\n\n\\** Ligatured results are mapped here **\\\n\\**\n\\char 10037 AHA_TINCO\n\\char 10038 HWESTA_TINCO\n\\char 10039 ANCA_CLOSED SILME_AHA\n**\\\n\n\\char 204A OLD_ENGLISH_AND\n\n\\char 200d ZWJ\n\n\\** Ligatured sequences used as one entity in modes **\\\n\\beg seq AHA_TINCO\n  AHA ZWJ TINCO\n\\end\n\n\\beg seq HWESTA_TINCO\n  HWESTA ZWJ TINCO\n\\end\n\n\\beg seq ANCA_CLOSED SILME_AHA\n  SILME ZWJ AHA\n\\end\n\n"
```

Comment:
Not sure what caused this file to be generated. It appears to be new. Maybe I ran the build incorrectly? I ran `ruby ./bin_dev/make_all.rb`